### PR TITLE
multi: Add DCRDEX onboarding step 1

### PR DESCRIPTION
**Pull request by ukane-philemon**
Part of #98 

Screenshot 2023-09-28 at 2 07 04 PM
Screenshot 2023-09-28 at 2 06 45 PM

Design:
There's a little difference in the inputs implemented. I thought it'd be better to utilize what we already have than reinvent the wheel with a minor difference. 
Screenshot 2023-09-28 at 2 08 35 PM

### CODE REVIEW

**Review by dreacot — COMMENTED** [review]
the layout looks distorted on my linux (although looks perfect on my mac)

Screenshot from 2023-09-28 08-35-53

below is more info on my machine

Screenshot from 2023-09-28 08-36-56

**Review by dreacot — COMMENTED** [review]
the layout looks distorted on my linux (although looks perfect on my mac)

Screenshot from 2023-09-28 08-35-53

below is more info on my machine

Screenshot from 2023-09-28 08-36-56

**Comment by ukane-philemon** [comment]
> the layout looks distorted on my linux (although looks perfect on my mac)
> 
> Screenshot from 2023-09-28 08-35-53
> 
> below is more info on my machine
> 
> Screenshot from 2023-09-28 08-36-56

Hmm, weird. Sadly I don’t have a Linux machine to test with. Do you know what could be the cause?

File: ui/values/dimensions.go [hunk 1]
```diff
@@ -118,4 +118,6 @@ var (
 	MobileAppHeight = unit.Dp(780)
 
 	StartMobileView = unit.Dp(600)
+
+	Length1000 = 1000
```
Comment (dreacot) [inline ui/values/dimensions.go:122]: @ukane-philemon this should be the culprit 
```suggestion
	Length1000 = unit.Dp(1000)
```

File: ui/page/dcrdex/dex_onboarding_page.go [hunk 2]
```diff
@@ -123,7 +123,7 @@ func (do *DEXOnboarding) Layout(gtx C) D {
 			// TODO: Add Step Select Server and Step Registration Fee
 			return do.Theme.List(do.scrollContainer).Layout(gtx, 1, func(gtx C, i int) D {
 				gtx.Constraints.Max = image.Point{
-					X: int(cryptomaterial.MaxWidth),
+					X: gtx.Dp(values.MarginPadding450),
```
Comment (ukane-philemon) [inline ui/page/dcrdex/dex_onboarding_page.go:126]: @dreacot, please can you retest on your device?

Comment (dreacot) [reply ui/page/dcrdex/dex_onboarding_page.go:126]: fixed

**Review by dreacot — APPROVED** [review]

**Pull request merged**